instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -131,10 +131,6 @@ def _instrumented_requests_call(
 
         url = remove_url_credentials(url)
 
-        labels = {}
-        labels[SpanAttributes.HTTP_METHOD] = method
-        labels[SpanAttributes.HTTP_URL] = url
```

Comment:
These changes don't seem directly related to the description of this change, were these attributes superfluous?

Reply:
I think this is the dead code @mariojonke is talking about in the PR description.

Reply:
Yes, the `labels` are a leftover from the old metrics implementation. Since all the metrics related code got removed, the labels are unused.